MAUI/DataGrid/Summaries.md

```diff
@@ -11,17 +11,647 @@ documentation: UG
+
+![DataGrid with customized caption summary row](Images\Caption-Summary\ShowSummaryInRow1.png)
+
+
+### Displaying summary for column
+
+Display summary information in the column by setting the `DataGridSummaryRow.ShowSummaryInRow` to `false` and define summary columns. `SfDataGrid.DataGridSummaryColumn` is the object of [DataGridSummaryRow.SummaryColumns]() collection that contains the following important properties:
+
+* [Name](): Defines name of the `DataGridSummaryColumn` to denote the `DataGridSummaryColumn` in `DataGridSummaryRow` with title.
+* [MappingName](): Defines the corresponding column name used for the summary calculation.
+* [SummaryType](): Defines the `SummaryType` (enum) property to define the aggregate type for the summary calculation. 
+
+The `DataGrid` control provides the following predefined aggregates:
+
+  * CountAggregate
+  * Int32Aggregate
+  * DoubleAggregate
+
+[CustomAggregate]() defines the `CustomAggregate` class object when the summary type is set as `Custom` that calculates the custom summaries.
+
+The [Format]() defines the `string` property that formats the summary value and displays it. The `Format` property may contains two parts separated by a colon (:). First part denotes the aggregate function name, and second part denotes display format of the summary value.
+
+Refer to the [Formatting Summary](#_Formatting_Summary) section to know more about how to format summary, and [Aggregate Types](#_Aggregate_Types) section to know about different summary types.
+
+In the following code snippet, summary is defined for `Salary` column:
+
+{% tabs %}
+{% highlight xaml%}
+<sfGrid:SfDataGrid.CaptionSummaryRow>
+    <sfGrid:DataGridSummaryRow Name="CaptionSummary" ShowSummaryInRow="False">
+        <sfGrid:DataGridSummaryRow.SummaryColumns>
+            <sfGrid:DataGridSummaryColumn Name="CaptionSummary"
+                                      Format="{}{Sum}"
+                                      MappingName="Salary"
+                                      SummaryType="DoubleAggregate" />
+        </sfGrid:DataGridSummaryRow.SummaryColumns>
+    </sfGrid:DataGridSummaryRow>
+</sfGrid:SfDataGrid.CaptionSummaryRow>
+{% endhighlight %}
+
+{% highlight c#%}
+DataGridSummaryRow summaryRow = new DataGridSummaryRow();
+summaryRow.ShowSummaryInRow = false;
+summaryRow.SummaryColumns.Add(new DataGridSummaryColumn()
+{
+    Name = "CaptionSummary",
+    MappingName = "Salary",
+    Format = "{Sum:c}",
+    SummaryType = SummaryType.DoubleAggregate
+});
+sfGrid.CaptionSummaryRow= summaryRow;
+{% endhighlight c#%}
+{% endtabs %}
+
+![DataGrid with caption summary cells customized](Images\Caption-Summary\ShowSummaryInColumn.png)
+
+N> The `CaptionSummaryColumn` text will be aligned based on the `DataGridColumn.TextAlignment`.
+
+### Caption summary template
+
+The data grid hosts any view(s) inside a caption summary for the entire row or for individual columns by loading a template.  
```

Comment:
The data grid allows you to host any view(s) inside a caption summary for the entire row or for individual columns by loading a template

Reply:
Modified.